user/views.py

```diff
@@ -1,26 +1,19 @@
-from django.http import HttpResponse
-from django.template.loader import get_template
+from django.contrib.auth import login, authenticate
+from django.shortcuts import render, redirect
 
-
-def login(request):
-    if request.method == 'GET':
-        template = get_template('login.html')
-        html = template.render()
-        
-        return HttpResponse(html)
-
-
-def logout(request):
-    if request.method == 'GET':
-        template = get_template('logout.html')
-        html = template.render()
-    
-        return HttpResponse(html)
+from user.forms import RegisterForm
 
 
 def register(request):
-    if request.method == 'GET':
-        template = get_template('register.html')
-        html = template.render()
-    
-        return HttpResponse(html)
\ No newline at end of file
+    if request.method == 'POST':
```

Comment:
It's so nice to see this long-form pattern for form processing is still remembered :)